tests/trainer/logging_/test_train_loop_logging.py

```diff
@@ -748,26 +748,29 @@ def validation_epoch_end(self, *_) -> None:
     trainer.fit(model, train_dataloaders=train_data, val_dataloaders=val_data)
 
 
-def test_no_batch_size_extraction_with_specifying_explictly(tmpdir):
-    batch_size = BoringModel().train_dataloader().batch_size + 1
+@mock.patch("pytorch_lightning.utilities.data.extract_batch_size")
+def test_no_batch_size_extraction_with_specifying_explictly(mock_method, tmpdir):
```

Comment:
This should also test logging a dictionary with

1. only metrics
2. a mix of tensors and metrics

Reply:
done!